src/oge/eia930.py

```diff
@@ -21,6 +21,155 @@
 logger = get_logger(__name__)
 
 
+# Dictionary mapping the new energy sources in EIA-930 to existing ones
+new_to_existing_energy_sources = {
+    "coal": "coal",
+    "gas": "gas",
+    "hydro": "hydro",
+    "hydro_excluding_pumped_storage": "hydro",
+    "pumped_storage": "hydro",
+    "nuclear": "nuclear",
+    "oil": "oil",
+    "battery_storage": "other",
+    "geothermal": "other",
+    "other": "other",
+    "other_energy_storage": "other",
+    "unknown_energy_storage": "other",
+    "solar": "solar",
+    "solar_w_integrated_battery_storage": "solar",
+    "solar_wo_integrated_battery_storage": "solar",
+    "unknown": "unknown",
+    "wind": "wind",
+    "wind_w_integrated_battery_storage": "wind",
+    "wind_wo_integrated_battery_storage": "wind",
+}
+existing_energy_source_names_to_codes = {
+    "coal": "COL",
+    "gas": "NG",
+    "hydro": "WAT",
+    "nuclear": "NUC",
+    "oil": "OIL",
+    "other": "OTH",
+    "solar": "SUN",
+    "unknown": "UNK",
+    "wind": "WND",
+}
+
+
+def convert_balance_data_to_gridemissions_format(year: int):
```

Comment:
Is this function called anywhere in the pipeline yet? I didn't see it

Reply:
Not yet. Will switch when testing the 2024 pipeline